docs/setup/k8s/running-in-production-k8s.md

```diff
@@ -172,3 +165,39 @@ The access keys or EC2 IAM policy must have the following permissions:
     ]
 }
 ```
+Common environment variables:
+```shell
+export AWS_REGION=<Your AWS region>
+export AWS_ACCOUNT=<Your AWS account ID>
+export ECR_PASSWORD=$(aws ecr get-login-password --region ${AWS_REGION})
+```
+Create the AWS credentials secret generated from `.aws/credentials` file configured with access key id and secret access key using the following command:
+```shell
+cat << EOF | kubectl --namespace nuclio create secret generic aws-credentials --save-config \
+--dry-run=client --from-file=credentials=/dev/stdin -o yaml | kubectl apply -f -
+[default]
+aws_access_key_id = ${AWS_ACCESS_KEY_ID}
+aws_secret_access_key = ${AWS_SECRET_ACCESS_KEY}
+EOF
+```
+> **Note:** This is needed to allow Kaniko creating the image repository prior to pushing the function image. 
+> Otherwise, Kaniko will fail to push the image to ECR because the image name is being determined during the build process.
+
+Create the ECR token secret to be used as `imagePullSecret` of function pods.
+Since ECR tokens stale after 12 hours, the secret must be refreshed periodically (can be done with a cron job as described in [Sergey's blog](https://skryvets.com/blog/2021/03/15/kubernetes-pull-image-from-private-ecr-registry/#update---aws-ecr-token-refresh))
```

Comment:
```suggestion
Since ECR tokens go stale after 12 hours, the secret must be refreshed periodically (can be done with a cron job as described in [this blogpost](https://skryvets.com/blog/2021/03/15/kubernetes-pull-image-from-private-ecr-registry/#update---aws-ecr-token-refresh))
```

Is Sergey a name we're all supposed to know?